easybuild/easyconfigs/c/cget/cget-0.1.6-foss-2018a-Python-3.6.4.eb

```diff
@@ -0,0 +1,57 @@
+##
+# This is an easyconfig file for EasyBuild, see https://github.com/easybuilders/easybuild
+# 
+# Author: Jonas Demeulemeester
+# The Francis Crick Insitute, London, UK
+##
+easyblock = 'Bundle'
+
+name = 'cget'
+version = '0.1.6'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://cget.readthedocs.io/en/latest/index.html'
+description = """Cmake package retrieval. This can be used to download and install cmake packages"""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+exts_defaultclass = 'PythonPackage'
+
+dependencies = [
+    ('Python', '3.6.4'),
+    ('CMake', '3.10.2'),
```

Comment:
@jdemeul Is `CMake` a *runtime* dependency here?

Better clarify that with a comment above this line, since that's really uncommon...